### ISSUE TITLE
when I use other network or loss，the val_acc is 0.00009 when train casia 

### ISSUE THREAD

**Issue by SuperZ2017** [Timodhau/Arcface-Keras-face_recognition#1]
when i use resnet50 and center loss
`
    class Dense_with_Center_loss(Layer):

        def __init__(self, output_dim, **kwargs):
            self.output_dim = output_dim
            super(Dense_with_Center_loss, self).__init__(**kwargs)

        def build(self, input_shape):
            self.kernel = self.add_weight(name='kernel',
                                      shape=(input_shape[1], self.output_dim),
                                      initializer='glorot_normal',
                                      trainable=True)
            self.bias = self.add_weight(name='bias',
                                    shape=(self.output_dim,),
                                    initializer='zeros',
                                    trainable=True)
            self.centers = self.add_weight(name='centers',
                                       shape=(self.output_dim, input_shape[1]),
                                       initializer='glorot_normal',
                                       trainable=True)

        def call(self, inputs):
            self.inputs = inputs
            return K.dot(inputs, self.kernel) + self.bias

        def compute_output_shape(self, input_shape):
            return (input_shape[0], self.output_dim)

        def loss(self, y_true, y_pred, lamb=0.5):
            y_true = K.cast(y_true, 'int32')
            crossentropy = K.categorical_crossentropy(y_true, y_pred, from_logits=True)
            centers = K.gather(self.centers, y_true[:, 0])  # get the center
            center_loss = K.sum(K.square(centers - self.inputs), axis=1)  # center loss
            return crossentropy + lamb * center_loss

    def create_model(backbone,lr):
    
        input_shape = (112,96, 3)
        inp = Input(shape=input_shape)
    
        for layer in backbone.layers:
            layer.trainable = True
        
        feature = backbone(inp)
        feature = GlobalAveragePooling2D()(feature) # 0.294
        feature = BatchNormalization()(feature)

        center_logits =Dense_with_Center_loss(num_classes)
        out = center_logits(feature)
        model = Model(inputs=inp, outputs=out)
        model.compile(loss=center_logits.loss, optimizer="Adam", metrics=['acc'])
    
        return model
`

or when I use mobilenet_V2 and arcface_loss ,they val_loss is 0.00009 and almost unchanged.
Can you gei me some Suggestions ?
